modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx

```diff
@@ -408,53 +402,139 @@ function Filters({
 		});
 	};
 
+	const getBreadcrumbItems = () => {
+		const breadcrumbItems: React.ComponentProps<
+			typeof ClayBreadcrumb
+		>['items'] = [
+			{
+				active: activeMode === FILTER_MODE.LIST ? true : false,
+				label: Liferay.Language.get('filters'),
+				onClick: () => {
+					setActiveMode(FILTER_MODE.LIST);
+				},
+			},
+		];
+
+		if (activeMode === FILTER_MODE.CREATION) {
+			let label = '';
+			if (activeFilterType === EFilterType.CLIENT_EXTENSION) {
+				label = Liferay.Language.get('new-client-extension-filter');
+			}
+			if (activeFilterType === EFilterType.DATE_RANGE) {
+				label = Liferay.Language.get('new-date-range-filter');
+			}
+			if (activeFilterType === EFilterType.SELECTION) {
+				label = Liferay.Language.get('new-selection-filter');
+			}
+
+			breadcrumbItems.push({
+				active: true,
+				label,
+			});
+		}
+
+		if (activeMode === FILTER_MODE.EDITION) {
+			breadcrumbItems.push({
+				active: true,
+				label: activeFilter!.label,
+			});
+		}
+
+		return breadcrumbItems;
+	};
+
 	return (
 		<ClayLayout.ContainerFluid>
-			<OrderableTable
-				actions={[
-					{
-						icon: 'pencil',
-						label: Liferay.Language.get('edit'),
-						onClick: onEdit,
-					},
-					{
-						icon: 'trash',
-						label: Liferay.Language.get('delete'),
-						onClick: onDelete,
-					},
-				]}
-				creationMenuItems={Object.keys(FILTER_TYPES).map((type) => ({
-					label: FILTER_TYPES[type as EFilterType].label,
-					onClick: () => onCreationButtonClick(type as EFilterType),
-				}))}
-				creationMenuLabel={Liferay.Language.get('new-filter')}
-				fields={[
-					{
-						label: Liferay.Language.get('name'),
-						name: 'label',
-					},
-					{
-						label: Liferay.Language.get('Field Name'),
-						name: 'fieldName',
-					},
-					{
-						label: Liferay.Language.get('type'),
-						name: 'displayType',
-					},
-				]}
-				items={filters}
-				noItemsButtonLabel={Liferay.Language.get('new-filter')}
-				noItemsDescription={Liferay.Language.get(
-					'start-creating-a-filter-to-display-specific-data'
-				)}
-				noItemsTitle={Liferay.Language.get(
-					'no-default-filters-were-created'
-				)}
-				onOrderChange={({order}: {order: string}) => {
-					updateFDSFiltersOrder({fdsFiltersOrder: order});
-				}}
-				title={Liferay.Language.get('filters')}
-			/>
+			<ClayBreadcrumb className="my-2" items={getBreadcrumbItems()} />
+
+			{activeMode === FILTER_MODE.CREATION && (
+				<ClayLayout.Sheet>
+					{activeFilterType && (
+						<FilterFormComponent
+							dataSet={dataSet}
+							dismiss={() => setActiveMode(FILTER_MODE.LIST)}
```

Comment:
I still see a lot of `dismiss`es in this PR, 22 search hits. Can we get rid of all of them?

Reply:
Well, they should handle the cancel button.

Reply:
After removing this `dismiss` call 
![image](https://github.com/user-attachments/assets/f2944c07-25e4-4b1e-bcb8-3a2558fe649e)

maybe it makes more sense to rename the `dismiss` to `onCancel`

Reply:
`onCancel` sounds good, if the event is in child component and we cannot set `activeMode` inline.